examples/example_tf_training_tfexample/main.py

```diff
@@ -30,32 +34,41 @@ def train(_):
 
     config = tf.estimator.RunConfig(tempfile.mkdtemp())
 
-    training_data_dir = get_data_dir("train")
-    feature_context = Datasets.get_context(training_data_dir)
+    train_data_dir = get_data_dir("train")
+    schema_path = os.path.join(train_data_dir, "_inferred_schema.pb")
 
-    (feature_names, label_names) = feature_context.multispec_feature_groups
-    features = [tf.feature_column.numeric_column(x) for x in feature_names]
+    feature_spec, _ = Datasets.parse_schema(schema_path)
+    # we use OrderedDict and sorted keys for features for determinism
+    all_features = OrderedDict([(name, tf.feature_column.numeric_column(name, default_value=.0))
+                                for name in sorted(feature_spec.keys())])
+    feature_columns = all_features.copy()
+    label_keys = sorted([l for l in set(feature_columns.keys()) if l.startswith("class_name")])
```

Comment:
So just to make sure I have this right - because of the switch to Example + schema.proto, we kinda lose the ability (at least at this point in the pipeline) to automatically parse features, labels separately without hardcoding the label name. This is because we don't (currently) encode this in the schema object like we did with tf record spec - even if featran was used to produce this dataset. Previously it was nice to just have them be split automatically based on the multispec id on the reading side, is this possible with schema? Because in the Featran case now there are essentially two places (one scala, one python) where we have to hardcode the label names.